resources/assets/less/bem/user-home-beatmap-list.less

```diff
@@ -0,0 +1,110 @@
+/**
+ *    Copyright 2015-2017 ppy Pty. Ltd.
+ *
+ *    This file is part of osu!web. osu!web is distributed with the hope of
+ *    attracting more community contributions to the core ecosystem of osu!.
+ *
+ *    osu!web is free software: you can redistribute it and/or modify
+ *    it under the terms of the Affero GNU General Public License version 3
+ *    as published by the Free Software Foundation.
+ *
+ *    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+ *    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *    See the GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+.user-home-beatmap-list {
+  @top: user-home-beatmap-list;
+
+  &__beatmap {
+    display: flex;
+    margin-bottom: 10px;
+    width: 100%;
+
+    border-radius: 5px;
+    .link-plain();
+    color: #232323;
+
+    transition: 0.1s;
+
+    &:hover {
+      background-color: lighten(#333, 10%);
+      color: white;
+    }
+
+    .own-layer();
+  }
+
+  &__cover {
+    flex-shrink: 0;
+    width: 70px;
+    height: 60px;
+    border-radius: 5px;
+    background-size: cover;
+    background-position: 50%;
+  }
+
+  &__meta {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 5px;
+    width: 100%;
+    padding: 0px 5px;
+    max-width: calc(100% ~'-' 105px);
+    color: white;
+    font-weight: 400;
+    font-style: italic;
+  }
+
+  &__title {
+    font-size: 14px;
+    margin-top: -2px;
+  }
+
+  &__artist {
+    font-size: 12px;
+    margin-top: -3px;
+  }
+
+  &__creator {
+    font-size: 10px;
+    margin-top: 2px
+  }
+
+  &__beatmap {
+    display: flex;
+    margin-bottom: 10px;
+    width: 100%;
+  }
+
+  &__cover {
+    flex-shrink: 0;
```

Comment:
I think `flex: none` the one you're looking for (idk it's slightly more obvious to me).